PocketKit/Sources/PocketKit/PocketAppDelegate.swift

```diff
@@ -216,4 +216,8 @@ public class PocketAppDelegate: UIResponder, UIApplicationDelegate {
         )
         Adjust.appDidLaunch(adjustConfig)
     }
+
+    public func trackWidgetOpen() {
```

Comment:
this name might not be totally appropriate, since what we are really tracking here is the app opened from the widget. Once we have deep links in place, it will track an article opened in the app from the widget

Reply:
So yes, if we change the call site (see this [comment](https://github.com/Pocket/pocket-ios/pull/779#discussion_r1238938299)) it might be worth storing into `UserDefaults`. Not sure that `sessionService` is the best place to do it. We could add a dedicated `WidgetTracker` object (injected in the `TimelineProvider` possibly`) that takes care of that

Reply:
sounds good, will rename to `trackWidgetTapped` and we can refactor when we have deep links. Let me know if you prefer a different name! naming is hard 🤣